src/systems/camera-system.js

```diff
@@ -409,14 +406,7 @@ export class CameraSystem {
         this.viewingRig.object3D.matrixWorld.decompose(position, quat, scale);
         position.setFromMatrixPosition(this.viewingCamera.matrixWorld);
         position.y = position.y - 1.6;
-        setMatrixWorld(
-          this.avatarRig.object3D,
-          new THREE.Matrix4().compose(
-            position,
-            quat,
-            scale
-          )
-        );
+        setMatrixWorld(this.avatarRig.object3D, new THREE.Matrix4().compose(position, quat, scale));
```

Comment:
Better to avoid to create a new matrix instance in tick (Sorry, this is an existing issue. Making another PR is ok.)

Reply:
Yeah this was just a formatting change. I see a few spots in here we are creating some garbage.